rest/route/github.go

```diff
@@ -462,6 +462,14 @@ func (gh *githubHookApi) handleMergeGroupChecksRequested(event *github.MergeGrou
 		"head_sha": event.GetMergeGroup().GetHeadSHA(),
 		"message":  "merge group received",
 	})
+	// Ensure that a project exists before creating an intent. Otherwise, intent creation will fail.
```

Comment:
I think it's very possible we'll look at this in the future and think, "'it's fine if intent creation fails", so we need to be clear that that'll always post an error to the user.